static/styles/portico/portico.css

```diff
@@ -1353,9 +1357,14 @@ input.new-organization-button {
 }
 
 @media (width <= 800px) {
+    .app {
+        overflow: hidden;
+    }
+
     .app.help {
         .markdown {
             width: 100vw;
+            overflow: auto;
```

Comment:
Actually, first I tried without this, but for better view in smaller devices this is needed. 
That is using html scrollbar in small screens too, but that actually disrupts the view as you can see in this picture
![image](https://user-images.githubusercontent.com/56730716/107129741-99b13f00-68ed-11eb-90f8-0c8166ef4e37.png)

Having the overflow auto for that element itself works better for smaller devices and touch-based also. So as you can see in this picture, even becoming small it will not disrupt the view.
![image](https://user-images.githubusercontent.com/56730716/107129900-c9147b80-68ee-11eb-9197-b291dc272b06.png)

And yes simplebar is completely removed from the right scrollbar.

Reply:
Sure to hear from you what feels better. We can do that way.

Reply:
I think the only problem in your “disrupted” screenshot is that the left sidebar is too narrow, right? This seems to be a problem with `100vw` vs. `100%`, and this fixes it:

```diff
--- a/static/styles/portico/portico.css
+++ b/static/styles/portico/portico.css
@@ -1353,15 +1353,10 @@ input.new-organization-button {
 }
 
 @media (width <= 800px) {
-    .app {
-        overflow: hidden;
-    }
-
     .app.help {
         .markdown {
             width: 100vw;
             margin-left: 0;
-            overflow: auto;
 
             .content {
                 max-width: none;
@@ -1383,7 +1378,7 @@ input.new-organization-button {
         .sidebar {
             position: fixed;
             top: 59px;
-            right: calc(100vw - 50px);
+            right: calc(100% - 50px);
             width: 100%;
             min-width: unset;
             height: calc(100vh - 59px);
```

Reply:
Cool! This works awesome. Got it, making this change quickly.